ls1intum/Artemis
# Programming-Exercise: Don't query test cases for student

### Checklist

### Motivation and Context



### Description


The get programming exercise test cases REST endpoint did not have user permissions.
This lead to a lot of access denied errors on Sentry...

The user impact is still very low as the error was caught and it should not have changed anything compared to before (the test case validation only improves the information if available, if not it is treated as before the test cases were integrated).

### Steps for Testing


1. Log in to ArTEMiS
2. Start a programming exercise for which you only have user permissions (enrolled in course)
3. Programming exercise test case get call should not return access denied

### Screenshots
